### ISSUE TITLE
Block commenting with keyboard shortcuts

### ISSUE THREAD

**Issue by tdrsam** [simonsteele/pn#88]
Just wondering if there is - or will be in later versions - a method for commenting blocks of code with a keyboard shortcut eg. Ctrl+(something)

**Comment by simonsteele** [comment]
You can set a keyboard shortcut for the comment menu items in the options dialog, and I think the default for comment is Ctrl+.

**Comment by tdrsam** [comment]
Hi Simon,

Thanks so much. I've got it sorted now.

Cheers.

Date: Fri, 24 Apr 2015 01:37:02 -0700

**Comment by tdrsam** [comment]
Hi Simon,
 
I hope you don't mind me contacting you out of the blue. I got a reply from you a while ago about block commenting in Programmers Notepad and I have another quick question.
 
I was writing some PHP and realized that I wanted the text editor to auto-complete my variable for me, and then I realized that I don't know if it can do that or not. I had a look around at some of the documentation but couldn't figure it out, so I wondered if I might be able to find out directly from the creator.
 
Basically, what I'm looking for is intelligent code completion, or what Microsoft calls Intellisense. Does PN have something like that?
 
Hope your well and the weather is better where you are than in Sydney.
 
Kind regards,
Samuel Martin.
